assets/src/components/nav/mobilePortraitNav.tsx

```diff
@@ -19,33 +19,37 @@ const MobilePortraitNav = ({
   const { mobileMenuIsOpen, routeTabs, openView, selectedVehicleOrGhost } =
     state
 
-  if (openView !== OpenView.None || selectedVehicleOrGhost) {
-    return (
-      <div className="m-nav--narrow m-nav--covered">
-        <div className="m-nav__app-content">{children}</div>
+  const isViewOpen = openView !== OpenView.None || selectedVehicleOrGhost
+
+  const navVisibilityStyle = isViewOpen ? "hidden" : "visible"
+
+  return (
+    <div className="m-nav--narrow">
+      <div
+        className="m-nav__nav-bar m-nav__nav-bar--top"
+        style={{ visibility: navVisibilityStyle }}
```

Comment:
I opted to do inline styling here as a work around for this issue w/ [jest toBeVisible ](https://github.com/testing-library/jest-dom/issues/209) not working with separate stylesheets unless there is some css configuration work done. If there is a preferred alternative approach I'd be open to trying it!

Reply:
My usual approach for these things has been to set a class based on the condition in question, and then just assert the presence / absence of the relevant classes on the element in my tests. This works, too, though, and `toBeVisible` actually tests the relevant property more directly.